crates/settlement-clients/starknet/src/tests/test.rs

```diff
@@ -0,0 +1,229 @@
+use super::{MadaraCmd, MadaraCmdBuilder};
+use crate::StarknetSettlementClient;
+use rstest::{fixture, rstest};
+use settlement_client_interface::SettlementClient;
+use starknet::{
+    accounts::{Account, ConnectedAccount, ExecutionEncoding, SingleOwnerAccount},
+    contract::ContractFactory,
+    core::types::{
+        contract::{CompiledClass, SierraClass},
+        BlockId, BlockTag, DeclareTransactionResult, Felt, FunctionCall, InvokeTransactionResult, StarknetError,
+        TransactionExecutionStatus, TransactionStatus,
+    },
+    macros::{felt, selector},
+    providers::{jsonrpc::HttpTransport, JsonRpcClient, Provider, ProviderError, Url},
+    signers::{LocalWallet, SigningKey},
+};
+use std::env;
+use std::path::Path;
+use std::sync::Arc;
+use std::time::Duration;
+use utils::settings::env::EnvSettingsProvider;
+use utils::settings::Settings;
+
+pub async fn spin_up_madara() -> MadaraCmd {
+    println!("Spinning up madara");
```

Comment:
instad of println! we can maybe use `log::trace` or remove it

Reply:
there are some println! pending, can you check

Reply:
No println logs now